### PR TITLE
Show how to get the system word size

### PR DESCRIPTION
**Pull request by nathanl**

### CODE REVIEW

File: system/doc/efficiency_guide/advanced.xml [hunk 1]
```diff
@@ -39,7 +39,9 @@
 
       <p>The unit of measurement is memory words. There exists both a
       32-bit and a 64-bit implementation. A word is therefore 4 bytes or
-      8 bytes, respectively.</p>
+      8 bytes, respectively. The value for a running system can be
+      determined by calling <seeerl marker="#system_info_wordsize">
```
Comment (garazdawi) [inline system/doc/efficiency_guide/advanced.xml:43]:
```suggestion
      determined by calling <seeerl marker="erts:erlang#system_info_wordsize">
```

**Comment by garazdawi** [comment]
Thanks! If you could just squash the two commits into one this is ready to be merged.

**Comment by nathanl** [comment]
@garazdawi squashed!

**Comment by nathanl** [comment]
There are other places where I think it would be useful to link within the docs. How would I build the docs locally to see that they look correct?

**Comment by nathanl** [comment]
Specifically I want to link to https://www.erlang.org/doc/apps/erts/garbagecollection#sizing-the-heap

**Pull request merged**